Update and rename Readme.md to onboarding.md

natalie-hub opened the pull request:
Replaced 1% placeholder. Added in: 
* Onboarding vision 
* Post-onboarding readiness
* Onboarding survey descriptions and process

<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
<!--
A description of what this pull request does.
-->

natalie-hub committed:
Update and rename Readme.md to onboarding.md

Replaced 1% placeholder. Added in: 
* Onboarding vision 
* Post-onboarding readiness
* Onboarding survey descriptions and process

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+
+### Post-Onboarding Readiness 
+
+Post Onboarding Readiness is an internal measure based on feedback throughout a new team member’s first 90 days at Mattermost. There are multiple milestones throughout the onboarding process where feedback will be solicited. This feedback intends to assess how willing and enabled a new colleague is to bring their best self to Mattermost as it relates to their role expectations, team and company f ...
```
```suggestion
Post-Onboarding Readiness is an internal measure based on feedback throughout a new team member’s first 90 days at Mattermost. There are multiple milestones throughout the onboarding process where feedback is solicited. This feedback assesses how willing and enabled a new colleague is to bring their best self to Mattermost as it relates to their role expectations, team and company focuses, and the Mattermost Leadership Principles. 
```

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+
+Post Onboarding Readiness is an internal measure based on feedback throughout a new team member’s first 90 days at Mattermost. There are multiple milestones throughout the onboarding process where feedback will be solicited. This feedback intends to assess how willing and enabled a new colleague is to bring their best self to Mattermost as it relates to their role expectations, team and company f ...
+
+Feedback will be requested throughout the 90-day Onboarding These consistent check-ins help new colleagues, HR and managers iterate with impact quickly. For example, one of the [V2MOM - Destination Workplace Measures](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f2ea8cda4_8_1852) is to have fewer than 5% questions repeat month-over-month. 
```
```suggestion
Feedback is requested from the new colleague's peers throughout the 90-day Onboarding. These consistent check-ins help new colleagues, HR, and managers iterate with impact quickly. For example, one of the [V2MOM - Destination Workplace Measures](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f2ea8cda4_8_1852) is to have fewer than 5% questions repeat month-over-month. 
```

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+
+Feedback will be requested throughout the 90-day Onboarding These consistent check-ins help new colleagues, HR and managers iterate with impact quickly. For example, one of the [V2MOM - Destination Workplace Measures](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f2ea8cda4_8_1852) is to have fewer than 5% questions repeat month-over-month. 
+
+#### Onboarding Feedback to assess post-onboarding readiness 
```
```suggestion
#### Onboarding Feedback to Assess Post-Onboarding Readiness 
```

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+TBD
+
+**New Staff Feedback Session with CEO and HR**
+Similar to [CEO listening tours](https://handbook.mattermost.com/operations/operations/company-cadence#ceo-listening-tours) new colleagues will be asked to share [likes and wishes](https://handbook.mattermost.com/company/about-mattermost/mindsets#likes-and-wishes) about anything Mattermost and Onboarding. 
```
```suggestion
Similar to [CEO listening tours](https://handbook.mattermost.com/operations/operations/company-cadence#ceo-listening-tours) in this session, new colleagues are asked to share [likes and wishes](https://handbook.mattermost.com/company/about-mattermost/mindsets#likes-and-wishes) about anything related to Mattermost and the Onboarding Program. 
```

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+Similar to [CEO listening tours](https://handbook.mattermost.com/operations/operations/company-cadence#ceo-listening-tours) new colleagues will be asked to share [likes and wishes](https://handbook.mattermost.com/company/about-mattermost/mindsets#likes-and-wishes) about anything Mattermost and Onboarding. 
+
+**New Colleague Feedback**
+The 90-Day Feedback provides a [360](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared) [shoulder check](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) for new team members, managers and [HR](https://handbook.mattermost.com/operations/workplace/people#team) and MLT to understand and clarify potential blinds ...
```
```suggestion
The 90-Day Feedback provides a [360](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared) [shoulder check](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) for new team members, managers, [HR](https://handbook.mattermost.com/operations/workplace/people#team), and MLT to understand and clarify potential blindspots as well as catch potential [brown M&Ms](https://handbook.mattermost.com/company/about-mattermost/mindsets#brown-m-and-ms) early on. 
```

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+**New Colleague Feedback**
+The 90-Day Feedback provides a [360](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared) [shoulder check](https://handbook.mattermost.com/company/about-mattermost/mindsets#shoulder-check) for new team members, managers and [HR](https://handbook.mattermost.com/operations/workplace/people#team) and MLT to understand and clarify potential blinds ...
+
+This is not a formal performance review. The intent is to align expectations and improve performance early on. Peer feedback is not anonymous. Managers may request additional context from the panel to help increase the new colleague's performance and post-onboarding readiness. Managers should evaluate the peer feedback against their own expectations of the new colelague's role. In some cases, exp ...
```
```suggestion
This is not a formal performance review. The intent is to align expectations and improve performance early on. Peer feedback is not anonymous. Managers may request additional context from the panel to help increase the new colleague's performance and Post-Onboarding Readiness. Managers should evaluate the peer feedback against their own expectations of the new colleague's role. In some cases, expectations may need to be reset with peers. Uncovering blindspots early on helps avoid future confusion or misaligned efforts. 
```

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+
+# Onboarding
+
+Welcome to Mattermost Onboarding! The goal of the Onboarding Program is to smoothly onboard and enable new colleagues into a safe and compliant working environment with a strong sense of belonging within their first 90 days at Mattermost. 
```
```suggestion
Welcome to Mattermost Onboarding! The goal of the Onboarding program is to welcome and enable new colleagues into a safe and compliant working environment with a strong sense of belonging within their first 90 days at Mattermost. 
```

natalie-hub replied:
Looks great!

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+**New Colleague 90-Day Feedback Process**
+
+ * **Managers**
+    * Complete a [new colleague peer panel form] for the new staff member. Notifications are sent to managers on a new colleague's date of employmnet. This will kick off the [review process](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared). 
```
```suggestion
    * Complete a [new colleague peer panel form] for the new staff member. Notifications are sent to managers on a new colleague's date of employment. This kicks off the [review process](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared). 
```
Missing a link for the new colleague peer panel form.

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+ * **Managers**
+    * Complete a [new colleague peer panel form] for the new staff member. Notifications are sent to managers on a new colleague's date of employmnet. This will kick off the [review process](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared). 
+    * Managers will recieve a unique link with peer panel results once the survey is sent. 
+    * People Ops will share the new colleague's self evaluation once completed. 
```
```suggestion
    * People Ops will share the new colleague's self-assessment once completed. 
```

natalie-hub replied:
Thank you for catching this!

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+    * Managers will recieve a unique link with peer panel results once the survey is sent. 
+    * People Ops will share the new colleague's self evaluation once completed. 
+ * **New Colleagues**
+    * Complete the [90-Day New Colleague Self Assessment](https://forms.gle/vyZeGCfG57BoGuhB9). Notifications are sent on day 75. 
```
```suggestion
    * Complete the [90-Day New Colleague Self-Assessment](https://forms.gle/vyZeGCfG57BoGuhB9). A notification is sent on day 75 with a reminder to complete the assessment. 
```

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+
+ * **Managers**
+    * Complete a [new colleague peer panel form] for the new staff member. Notifications are sent to managers on a new colleague's date of employmnet. This will kick off the [review process](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared). 
+    * Managers will recieve a unique link with peer panel results once the survey is sent. 
```
```suggestion
    * Managers receive a unique link with peer panel results after the survey is sent. 
```

justinegeffen commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,44 @@
+ * **Managers**
+    * Complete a [new colleague peer panel form] for the new staff member. Notifications are sent to managers on a new colleague's date of employmnet. This will kick off the [review process](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50#how-is-feedback-shared). 
+    * Managers will recieve a unique link with peer panel results once the survey is sent. 
+    * People Ops will share the new colleague's self evaluation once completed. 
```
```suggestion
    * People Ops share the new colleague's self-assessment once completed. 
```

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen reviewed (dismissed):
Thanks, @natalie-hub! I've committed some changes to tense and voice, as well as capitalization changes. The remaining suggestions require your input, but otherwise it looks good to me so far. :)

natalie-hub committed:
Update contributors/onboarding/onboarding.md

Co-Authored-By: Justine Geffen <justinegeffen@users.noreply.github.com>

natalie-hub committed:
update onboarding.md

Sorry, I added in additional links to Survey Monkey so people can access surveys at any time prior to notifications. Also, Lynn is going to create a zapier link to Bamboo's notification so that managers and new colleague's can be notified via Mattermost. I updated that as part of the process, too.

natalie-hub reviewed (commented):
I updated a few links under the New Colleague 90-Day Feedback Process

natalie-hub commented:
Thanks @justinegeffen - sorry if I messed up the order or did something wrong.  I intended to approve all of the edits that you made - thank you, again btw. I also then updated a few of the links and instruction in the process. Should I have saved that for a new PR?

justinegeffen committed:
Update contributors/onboarding/onboarding.md

justinegeffen commented:
Nope, nothing was messed up - all good! I don't think a new PR was needed, especially if the changes were as a result of/triggered by my suggestions. :)

hannaparks commented on contributors/onboarding/onboarding.md:
```diff
@@ -0,0 +1,45 @@
+
+Welcome to Mattermost Onboarding! The goal of the Onboarding program is to welcome and enable new colleagues into a safe and compliant working environment with a strong sense of belonging within their first 90 days at Mattermost. 
+
+Onboarding is an all-company partnership between a new colleague and Mattermost. Specifically, a new colleague’s manager, functional and working teams, onboarding [peers](https://docs.google.com/document/d/1DQULEvgOhGeEGLTzdOlSvo858vo94zmnMN8sygi-lHk/edit?ts=5e21f202), [HR](https://handbook.mattermost.com/operations/workplace/people#team) and MLT. 
```
Onboarding is an all-company partnership between a new colleague and Mattermost. Specifically, a new colleague’s manager, functional and working teams, onboarding [peers](https://docs.google.com/document/d/1DQULEvgOhGeEGLTzdOlSvo858vo94zmnMN8sygi-lHk/edit?ts=5e21f202), [HR](https://handbook.mattermost.com/operations/workplace/people#team) and [MLT](https://handbook.mattermost.com/operations/workplace/groups).

natalie-hub replied:
Thanks @hannaparks!
